client/components/SignIn.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class SignIn extends Component {
+  constructor() {
+    super();
+
+    this.initialState = {
```

Comment:
If you're going assign the value of this to `this.state`, why not just name it `this.state` instead of `this.initialState`